packages/react/src/feature-link-list/FeatureLinkListItem.tsx

```diff
@@ -63,11 +68,16 @@ export const FeatureLinkListItem = ({
 									left: 0,
 								},
 							}}
+							{...linkProps}
```

Comment:
I would much rather just see the simpler`aria-describedby={description ? descriptionId : undefined}`